core/src/main/java/org/mskcc/cbio/portal/util/StringUtilsLevenshtein.java

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
is Apache Commons not already accessible from the project? Can you not just import this method directly from Apache Commons, instead of adding this file in?